src/yoroi-wallets/types/types.ts

```diff
@@ -57,3 +58,18 @@ export type YoroiAmount = {
 export type YoroiMetadata = {
   [label: string]: string
 }
+
+export type YoroiNFT = {
```

Comment:
@MichalSzorad please sync with @wolverineks, leaning towards `YoroiToken['type'] = 'nft' | 'ft' | 'primary'` and maybe it's overlapping with his impl on the `AssetList`.